src/NzbDrone.Core/MediaFiles/EpisodeImport/DetectSample.cs

```diff
@@ -43,6 +43,19 @@ public DetectSampleResult IsSample(Series series, string path, bool isSpecial)
                 _logger.Debug("Skipping sample check for .strm file");
                 return DetectSampleResult.NotSample;
             }
+            
+            string[] videoExtensions = {".webm", ".mkv", ".flv", ".mts", ".m2ts", ".mt", ".mov", ".wmv", ".amv", ".mp4", ".mpg", ".mpeg", ".mpv", ".m4v", "3gp", "3g2", ".flv", ".avi"};
```

Comment:
This list doesn't cover the full list of extensions that Sonarr supports. What about extra files that Sonarr supports that aren't media files?